docs/source/markdown/podman.1.md

```diff
@@ -217,11 +217,11 @@ the exit codes follow the `chroot` standard, see below:
 
 ## FILES
 
-**libpod.conf** (`/usr/share/containers/libpod.conf`)
+**containers.conf** (`/usr/share/containers/containers.conf`)
 
-    libpod.conf is the configuration file for all tools using libpod to manage containers, when run as root.  Administrators can override the defaults file by creating `/etc/containers/libpod.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/libpod.conf` is created and replaces some fields in the system configuration file.
+    containers.conf is the configuration file for container engines.  Administrators can override the defaults file by creating `/etc/containers/containers.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/containers.conf` is created and replaces some fields in the system configuration file.
```

Comment:
```suggestion
    containers.conf is the configuration file for container engines, when run as root.  Administrators can override the defaults file by creating `/etc/containers/containers.conf`.  When Podman runs in rootless mode, the file `$HOME/.config/containers/containers.conf` is created and replaces some fields in the system configuration file.
```

Reply:
@TomSweeneyRedHat How about the new wording.